doc/src/manual/style-guide.md

```diff
@@ -123,7 +123,7 @@ Types such as `Union{Function,AbstractString}` are often a sign that some design
 When creating a type such as:
 
 ```julia
-type MyType
+struct MyType
```

Comment:
True, though I don't see that as being a problem. I can just make it `mutable struct` if you think that's better.